mlir/utils/performance/results.txt

```diff
@@ -0,0 +1 @@
+pmc: LDSBankConflict
```

Comment:
Can you add a bit of comments explaining how this work, why this file is needed, etc..?

Reply:
Also, I would call this file `rocmlir_metrics.txt` or something like this

Reply:
Yes I will add the explaining for that.